src/backend/metal/src/command.rs

```diff
@@ -1577,8 +1580,10 @@ impl RawCommandQueue<Backend> for CommandQueue {
                                     cmd_buffer
                                 });
                             journal.record(&*cmd_buffer);
-                            if STITCH_DEFERRED_COMMAND_BUFFERS {
+                            if self.stich_deferred {
                                 deferred_cmd_buffer = Some(cmd_buffer);
+                            } else {
+                                cmd_buffer.commit();
```

Comment:
Previously a bug?

Reply:
yes :) it took me a bit of head scratching to figure out